lib/csv/row.rb

```diff
@@ -0,0 +1,384 @@
+# frozen_string_literal: true
+# = csv.rb -- CSV Reading and Writing
```

Comment:
This file isn't `csv.rb`.
We can remove the header description by moving the description to README.